charts/azuremonitor-containers/templates/omsagent-deployment.yaml

```diff
@@ -33,6 +33,16 @@ spec:
    serviceAccountName: omsagent
   {{- end }}
    containers:
+{{- if and (ne .Values.Azure.Cluster.ResourceId "<your_cluster_id>") (.Values.omsagent.useAADAuth) }}
```

Comment:
indentation issue

Reply:
This is not indention issue as discussed.